infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

```diff
@@ -176,6 +216,25 @@ private OpenshiftRoleBinding createExecRoleBinding() {
         .build();
   }
 
+  private OpenshiftRoleBinding createStopWorkspacesRoleBinding() {
+    return new OpenshiftRoleBindingBuilder()
+        .withNewMetadata()
+        .withName(serviceAccountName + "-stop")
+        .withNamespace(projectName)
+        .endMetadata()
+        .withNewRoleRef()
+        .withName("workspace-stop")
+        .withNamespace(projectName)
+        .endRoleRef()
+        .withSubjects(
+            new ObjectReferenceBuilder()
+                .withKind("ServiceAccount")
+                .withName("che")
+                .withNamespace("che")
```

Comment:
Namespace is configurable. Use the `projectName` field.

Reply:
`projectName` and `serviceAccountName` refer to the user namespace and the user serviceaccount.  The `che` ServiceAccount in the `che` namespace need to be able to manage resources in a user namespace.

Reply:
yeah, here we need to provide a namespace where che-server is deployed. Not sure if the SA name is configurable though cc: @davidfestal

Reply:
it looks like we need to create a `@Singleton` that would be bound in the `OpenShiftInfraModule` / `KubernetesInfraModule` and expose information about the namespace / sa. 

My observations are that `che` SA name is hardcoded and not configurable and namespace name can be obtained via `System.getenv` (Need to check if this will work in all the cases e.g.:
- the operator uses `KUBERNETES_NAMESPACE` env var
- helm `POD_NAMESPACE`